mix.exs

```diff
@@ -5,9 +5,9 @@ defmodule SignsUi.Mixfile do
     [
       app: :signs_ui,
       version: "0.0.1",
-      elixir: "~> 1.4",
```

Comment:
I don't think it was necessarily wrong, but certainly outdated. There doesn't seem to be a lot of documentation out there on this specific field (one of the downsides to a more niche language like Elixir), but I think this line basically means "use any version of Elixir that is compatible with Elixir 1.4", which I suppose suggests that Elixir 1.11 is backwards compatible with Elixir 1.4 since that's what Signs UI was running on. I'm not 100% sure if that interpretation is correct though